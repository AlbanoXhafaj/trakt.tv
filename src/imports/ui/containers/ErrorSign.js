import React from "react";

//a red cycle with a big X in middle
const ErrorSign = () =>(
  <div className ='error-circle'>
    <div>X</div>
  </div>
);

export default ErrorSign;
