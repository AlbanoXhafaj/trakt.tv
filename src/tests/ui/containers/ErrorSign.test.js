import React from "react";

import "../../setups/setup_enzyme.js";
import { shallow , mount } from "enzyme";
import toJSON from 'enzyme-to-json';

import ErrorSign from "containers/ErrorSign";

describe("<ErrorSign />" , () => {

  const wrapper = shallow(<ErrorSign />);

  it("should match with the snapshot" , () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  })

});
