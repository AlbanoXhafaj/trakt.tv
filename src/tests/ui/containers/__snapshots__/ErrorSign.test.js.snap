// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorSign /> should match with the snapshot 1`] = `
<div
  className="error-circle"
>
  <div>
    X
  </div>
</div>
`;
